src/main/java/com/codeborne/selenide/appium/AppiumNavigator.java

```diff
@@ -0,0 +1,37 @@
+package com.codeborne.selenide.appium;
+
+import static com.codeborne.selenide.appium.WebdriverUnwrapper.instanceOf;
+
+import com.codeborne.selenide.Configuration;
+import com.codeborne.selenide.logevents.SelenideLogger;
+import io.appium.java_client.AppiumDriver;
+import io.appium.java_client.android.AndroidDriver;
+import io.appium.java_client.ios.IOSDriver;
+import java.util.HashMap;
+import java.util.function.Supplier;
+import javax.annotation.ParametersAreNonnullByDefault;
+
+@ParametersAreNonnullByDefault
+public class AppiumNavigator {
+
+  public void launchApp(Supplier<Runnable> driverSupplier) {
+    Configuration.browserSize = null;
+    Configuration.pageLoadTimeout = 0;
+    SelenideLogger.run("launch app", "", driverSupplier.get());
+  }
+
+  public void terminateApp(AppiumDriver driver, String appId) {
+    SelenideLogger.run("terminate app", appId, () -> {
+      HashMap<String, String> params = new HashMap<>();
```

Comment:
I usually declare variable as `Map<String, String> params`, not `HashMap`.

Reply:
Agree will change it.